src/app/dashboard/rankingTable/RankingTable.utils.ts

```diff
@@ -0,0 +1,5 @@
+export const capitalizeFirstLetter = (word: string) => {
+  const capitalizedWord = word.charAt(0).toUpperCase() + word.slice(1);
+
+  return capitalizedWord === 'Micha' ? 'Michał' : capitalizedWord;
```

Comment:
Podbijam - sprawdź czy na pewno dobrze Ci działa response camel to pascal interceptor 😁